autorest/models/code_model.py

```diff
@@ -52,7 +52,8 @@ def get_python_type(self, namespace=None):
 class CodeModel:
 
     def __init__(self):
-        self.client_name = None
+        self.python_client_name = None
+        self.camel_case_client_name = None
```

Comment:
What's the difference?

Reply:
Python client name is snake case client name (i.e. client_name), while camel case is ClientName